Code/Editor/Lib/Tests/test_ViewportManipulatorController.cpp

```diff
@@ -77,7 +77,7 @@ namespace UnitTest
     class ViewportManipulatorControllerFixture : public AllocatorsTestFixture
     {
     public:
-        static const AzFramework::ViewportId TestViewportId = AzFramework::ViewportId(0);
+        static const AzFramework::ViewportId TestViewportId;
```

Comment:
`ViewportId` is just an `int`, so you could use a `constexpr inline` variable instead:

```diff
- static const AzFramework::ViewportId TestViewportId;
- const AzFramework::ViewportId ViewportManipulatorControllerFixture::TestViewportId = AzFramework::ViewportId(0);
+ static constexpr inline AzFramework::ViewportId TestViewportId{0};
```

Reply:
That's a good shout, I can update that in a follow-up PR 👍